### ISSUE TITLE
[FEAT]: Website

### ISSUE THREAD

**Issue by LelouchFR** [LelouchFR/skill-icons#101]
### Description

The point will be to be able to pick the order in the way the icons should show up, and be able to customize a little bit anything about it.
It would have a homepage which tells the story about the project, show contributor on the page, and some insights about the api and some other things.
You could click on a button to generate a link, copy to clipboard, and other customisations.

### Additional Information

Here is the figma file on which is going to get worked on (ask for permission if you want to contribute to the file): https://www.figma.com/design/ZsJxrxKOxXN8Hry65uAnxc/Skill-Icons-Website?node-id=0-1&t=UwXYf5yHxZ0dkAKE-1

### Code of Conduct Acknowledgment

- [X] I have read and agree to the Code of Conduct.

**Comment by Francois-lenne** [comment]
@LelouchFR  if you are a student you can have figma premium for free : 

https://help.figma.com/hc/en-us/articles/360041061214-Verify-education-status

**Comment by LelouchFR** [comment]
> @LelouchFR if you are a student you can have figma premium for free :
> 
> https://help.figma.com/hc/en-us/articles/360041061214-Verify-education-status

well... the thing is that I'm still in lycée (terminal), so I'm not technically a student :/ so I just have 3 figma files, and they are already used at there whole potential, this is sad.

**Comment by LelouchFR** [comment]
@Francois-lenne I found out I could just create a new team :P so It's all good I guess, here is the link: https://www.figma.com/design/ZsJxrxKOxXN8Hry65uAnxc/Skill-Icons-Website?node-id=0-1&t=UwXYf5yHxZ0dkAKE-1

**Comment by ItsJoshBrown** [comment]
A website would be cool. 

are you meaning, you want to set something like this set up? 

tandpfun/skill-icons/issues/101

https://www.skilliconsgenerator.com/
https://svelte.dev/repl/e29cb982e5df4cbc8b80fdc9ad04cbe8?version=3.48.0

for like an icon picker etc?

**Comment by eng-gabrielscardoso** [comment]
@ItsJoshBrown yes, but more I guess, a complete homepage describing the project about, history, how to contribute, a icon picker (how you mentioned), etc. We could map the features that the website could have, including this

**Comment by LelouchFR** [comment]
@ItsJoshBrown It would be one of the parts of the website, so people can make their icons as they want, with all the possible ways to modify their link (in plane text, if you know what I mean). Also another part that would be cool is link a simple homepage to direct people on the repo and the socials (I think it could be interesting to open a discord server) and a simple way to display what the project is about.

**Comment by knightmar** [comment]
Just to warn you : the link of the repo is leading to a 404 error : https://go-skill-icons.vercel.app/
idk if the url changed

**Comment by ghost** [comment]
@knightmar it's not really an error, it's just a not found 404 error page, cause there is nothing to display on that route, the main route of the api is https://go-skill-icons.vercel.app/api/icons?i={icon_id}

**Comment by knightmar** [comment]
Yeap I know, but if there is no main page, don't put the link in the repo description, it may be confusing.
(Or just put a basic main page with a link to this repo)

**Comment by ghost** [comment]
When the site is released, the link will be replaced, i know it's confusing (patience is the key to relief)

**Comment by LelouchFR** [comment]
> Yeap I know, but if there is no main page, don't put the link in the repo description, it may be confusing.
> (Or just put a basic main page with a link to this repo)

The thing is that I didn't even put the link there, vercel at some point decided to put that here :P I didn't tried to changed but it will be functional after the creation of the website

**Comment by sluxzer** [comment]
I would love to help, where I can start?

Also, if you don't mind, It would be great if we create grouping

Ex: 
- OS: Windows, Apple, Ubuntu, Linux
- IDE: Sublime, VS Code
- Project Management: Notion, JIRA, Asana, Trello, ClickUp
- Documentation: Postman, Swagger, Storybook

**Comment by ItsJoshBrown** [comment]
@LelouchFR  what would be a good way to get in contact with you? I wanted to show/ share something with you get your thoughts on for a website.  Do you have discord? i added you on twitter but didn't let me message you right then and there.

**Comment by LelouchFR** [comment]
> @LelouchFR  what would be a good way to get in contact with you? I wanted to show/ share something with you get your thoughts on for a website.  Do you have discord? i added you on twitter but didn't let me message you right then and there. 

You can contact me on [Instagram](https://www.instagram.com/gumball0670.rs), else on [LinkedIn](https://www.linkedin.com/in/baptiste-zahnow). I could provide you my discord there if needed.

**Comment by ItsJoshBrown** [comment]
> > @LelouchFR  what would be a good way to get in contact with you? I wanted to show/ share something with you get your thoughts on for a website.  Do you have discord? i added you on twitter but didn't let me message you right then and there.
> 
> You can contact me on [Instagram](https://www.instagram.com/gumball0670.rs), else on [LinkedIn](https://www.linkedin.com/in/baptiste-zahnow). I could provide you my discord there if needed.

Message sent on insta as well as FR I think it has to be accepted before the message will send? idk

**Comment by LelouchFR** [comment]
Hello @sluxzer @ItsJoshBrown @eng-gabrielscardoso @knightmar 

I've created a Google Forms to get informations about the api to know what I could add into the website. If you want to help me to have content for the website (I mean, with the results of the form, I could do something like comments to put into a comment section)

Here is the link of the google form:

https://forms.gle/SC925eRJg19wXxKL6

**Comment by evgo2017** [comment]
Hi @LelouchFR and everyone! 👋

I saw this issue and realized I recently built an open-source tool that implements exactly what is described here! 

I created **[Skill Icons Picker](https://evgo2017.com/project/skill-icons-picker)**. It's an interactive website that allows you to:
- 🖱️ **Browse & Search** through all the current icons quickly.
- 🔄 **Drag and Drop** to pick and arrange the order of the icons exactly how you want them.
- 🎨 **Customize** settings like Light/Dark theme and icons per line.
- 📋 **Live Preview & 1-Click Copy**: Instantly see life-like previews and copy the generated Markdown code to your clipboard for your GitHub profile.
- 📖 **Icon Meanings & Tooltips**: Hover over any icon to see its exact technology name and description (helpful for discovering new tools!).

The backend is fully automated with a daily GitHub Action that syncs the latest SVG files directly from this repository, so the icon list is always up to date.

Here is the live demo: https://evgo2017.com/project/skill-icons-picker or https://evgo2017.github.io/skill-icons-picker/
And the open-source repository (built with Vite & Vanilla JS): https://github.com/evgo2017/skill-icons-picker

Feel free to check it out! Let me know if you consider linking this in the README or if there are any specific elements you'd love to see integrated. I'd be more than happy to collaborate! 🚀

<img width="1583" height="1321" alt="Image" src="https://github.com/user-attachments/assets/290e8cc1-50ff-4072-bc8b-66e0ff9543ab" />

**Comment by LelouchFR** [comment]
Hi @evgo2017, thank you for having created a website, but we already have one in production. maybe it doesn't have the same features as yours, but the official one will stay to be https://go-skill-icons.netlify.app

You can let a comment about how you find the icons at the [google form](https://forms.gle/SC925eRJg19wXxKL6) to be mentioned on the website

About the codebase of the website, I will do my best to open source it in not too long, just have a lot of things to do right now.

**Comment by evgo2017** [comment]
@LelouchFR Thanks for the reply. The official website looks great, and it includes one of the features I wanted most — not having to manually write the URL, which is really useful. I think the site is already polished enough to be added to the repository details.

I’ve already filled out the form, and I’m also looking forward to seeing the official website code open-sourced whenever you have the time. Thanks again for maintaining such an awesome project!

<img width="1560" height="1229" alt="Image" src="https://github.com/user-attachments/assets/c7266068-c78d-4cdb-a44d-5f2e4a5b9acf" />

**Comment by ItsJoshBrown** [comment]
This is hilarious, I had no idea this was live. I actually repurposed Trixzyy's repo that he did for the old skill-icons repo
 and built a website around it as well https://skills-builder.vercel.app/home

Since that's the case I'll probably take this one done soon then.

The website looks pretty good @LelouchFR
